cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -650,6 +656,56 @@ private void setAndTestRuntimeArgs(String namespace, String appId, String runnab
 
   @Category(XSlowTests.class)
   @Test
+  public void testMultipleWorkflowInstances() throws Exception {
```

Comment:
As discussed, changed the test case to compare the runs of the program instead of comparing the StartTime.